scripts/generate_compile_commands.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
```

Comment:
Should be `python2` or `python3`

Reply:
Please provide a PR to https://github.com/mapbox/node-cpp-skel if you think this needs fixed. We use `#!/usr/bin/env python` currently since this works on: a) latest OS X and b) ubuntu precise/trusty.

Does this break with latest ubuntu? How can we support all systems here?

Reply:
PR is done upstream, and I've corrected this script to work with both Python2 and 3.

More details on current state of what `python` means is detailed in that PR: https://github.com/mapbox/node-cpp-skel/pull/120#issuecomment-384404925